elasticsearch-core/src/main/scala/com/sumologic/elasticsearch/restlastic/dsl/MappingDsl.scala

```diff
@@ -93,6 +97,25 @@ trait MappingDsl extends DslCommons {
     }
   }
 
+  // Supported in elasticsearch v2.4
```

Comment:
I think we need a formalized way of handling elasticsearch 2.x stuff. Ideally it would be a different DSL you could import or something. Another task to think about pre-2.0.